### PR TITLE
Add redisinsight v1.13.1 release notes.

### PR DESCRIPTION
**Pull request by GnaneshKunal**

### CODE REVIEW

**Comment by GnaneshKunal** [comment]
@nermiller Please review.

I'll let you know once RedisInsight v1.13.1 is released.

**Comment by nermiller** [comment]
sounds good, @GnaneshKunal!

File: content/ri/release-notes/v1.13.0.md [hunk 1]
```diff
@@ -6,6 +6,19 @@ description: RedisInsight v1.13.0
...
 ---
 
+## 1.13.1 (November 2022)
+
+This is the maintenance release of RedisInsight 1.13 (v1.13.1)!
+
+### Fixes:
+- Core:
+  - Fixed container vulnerabilities.
+  - Disable healthcheck API to overload RedisInsight DB.
```
Comment (nermiller) [inline content/ri/release-notes/v1.13.0.md:16]: Mind explaining this fix a bit further? What was happening?

Comment (GnaneshKunal) [reply content/ri/release-notes/v1.13.0.md]: done

**Comment by GnaneshKunal** [comment]
@nermiller 

Redisinsight has been released. Once its fine, please merge it.

File: content/ri/release-notes/v1.13.0.md [hunk 2]
```diff
@@ -6,6 +6,19 @@ description: RedisInsight v1.13.0
...
 ---
 
+## 1.13.1 (November 2022)
+
+This is the maintenance release of RedisInsight 1.13 (v1.13.1).
+
+### Fixes:
+- Core:
+  - Fixed container vulnerabilities.
+  - Disable healthcheck API to overload RedisInsight DB. Earlier a separate session was created for each healthcheck hit which overloaded the database with too many session tokens. Now healtcheck API doesn't create any session token.
```
Comment (nermiller) [inline content/ri/release-notes/v1.13.0.md:16]:
```suggestion
  - Prevented healthcheck API from overloading RedisInsight DB. Earlier, a separate session was created for each healthcheck hit, which overloaded the database with too many session tokens. Now, healtcheck API doesn't create any session tokens.
```

**Comment by nermiller** [comment]
@GnaneshKunal left you one last comment. As soon as you clear it, I'll merge and publish. Thank you!

**Pull request merged**